src/utils/helper-functions/versions.ts

```diff
@@ -0,0 +1,23 @@
+import * as semver from 'semver'
+
+let ngrxEffectsVersion: string | undefined
+function getNgrxEffectsVersion(): string {
+  if (ngrxEffectsVersion) return ngrxEffectsVersion
+
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const effects = require('@ngrx/effects/schematics-core') as {
+      platformVersion: string
+    }
+    // version in file is ^x.x.x, but this isn't valid for semver
+    ngrxEffectsVersion = effects.platformVersion?.substr(1)
+  } catch (err) {
```

Comment:
```suggestion
  } catch {
```

Reply:
Aha, I missed that by default `caughterrors` is set to `none`: https://eslint.org/docs/rules/no-unused-vars#caughterrors